recipe/README.md

```diff
@@ -0,0 +1,46 @@
+To build a conda pytorch package with GPU support you can use docker and the
+`build-locally.py` script.
+
+1. Install docker. Ensure that the following command succeeds:
+
+```bash
+docker run hello-world
+```
+
+2. Build a specific version with the command
+```bash
+python build-locally.py
+```
+
+3. Generally speaking, this package takes too long to compile on any of our CI
+  resources. One should follow CFEP-03 to package this feedstock.
+
+The following script may help build all cuda version sequentially:
+```bash
+#!/usr/env/bin bash
+
+set -ex
+conda activate base
+
+docker system prune --force
+configs=$(find .ci_support/ -type f -name '*cuda_compiler_version[^nN]*' -printf "%p ")
```

Comment:
Shouldn't we just build all linux versions here? They also timeout quite often.

Reply:
Ok, I updated the command.